apps/etterlatte-beregning/src/main/kotlin/model/BeregningService.kt

```diff
@@ -22,14 +23,47 @@ import java.time.YearMonth
 import java.util.*
 
 // TODO hvordan håndtere vedtakstidspunkt?
-class BeregningService {
+class BeregningService(
+    private val beregningRepository: BeregningRepository,
+    private val vilkaarsvurderingKlient: VilkaarsvurderingKlient
+) {
+
+    fun hentBeregning(behandlingId: UUID): Beregning = beregningRepository.hent(behandlingId)
+
+    fun bekreftberegnetresulat(behandlingId: UUID): Beregning {
+        val beregning: Beregning = beregningRepository.hent(behandlingId)
+        return beregningRepository.lagre(beregning)
+    }
+
+    suspend fun lagreBeregning(behandlingId: UUID, accessToken: String): Beregning {
```

Comment:
Ville kanskje lagt til tester for de nye funksjonene.

Reply:
Jepp vi skal lage tester, har egen task i jira for det 😎 Men siden det ikke blir i bruk fra start, så vil vi ha ut koden fotr å få mindre pr-er